services-js/commissions-app/src/server/email/EmailContent.test.ts

```diff
@@ -0,0 +1,33 @@
+import EmailContent from './EmailContent';
+
+describe('Renders email body', () => {
+  const emailContent = new EmailContent();
+  const applicationData = {
+    commissions: ['Board 1', 'Board 3', 'Board 5'],
+    email: 'x@y.com',
+    newId: '23',
+  };
+
+  const applicantEmailBodies = emailContent.renderApplicantBodies(
+    applicationData
+  );
+  const policyOfficeEmailBodies = emailContent.renderPolicyOfficeBodies(
+    applicationData
+  );
+
+  it('applicant HTML', () => {
```

Comment:
If you feel like being grammatical, you can change these `it`s to `test`s.

Reply:
...you know me so well! 😄